src/foam/nanos/u2/navigation/SideNavigation.js

```diff
@@ -0,0 +1,147 @@
+foam.CLASS({
+    package: 'foam.nanos.u2.navigation',
+    name: 'SideNavigation',
+    extends: 'foam.u2.View',
+  
+    documentation: 'Side navigation bar',
+  
+    implements: [
+        'foam.mlang.Expressions'
+      ],
+       requires: [
+        'foam.nanos.menu.Menu'
+      ],
+       imports: [
+        'menuDAO'
+      ],
+       css: `
+        ^ .side-nav {
+          height: 100%;
+          width: 200px;
+          position
+          z-index: 1;
+          top: 0;
+          left: 0;
+          overflow-x: hidden;
+          background-color: white;
+          display: inline-block;
+        }
+        ^ .nav-row {
+          display: block;
+        }
+        ^ .side-nav a {
+          display: inline-block;
+          margin: 8px 8px 8px 8px;
+          text-decoration: none;
+          font-size: 20px;
+          transition: 0.3s;
+        }
+        ^ .side-nav a:hover {
+          color: #f1f1f1;
+        }
+        ^ .accordion-card a {
+          font-size: 16px;
+          margin: 8px 8px 8px 50px;
+        }
+        ^ .accordion-card-hide {
+          display: none;
+        }
+        ^ .accordion-card-show {
+          display: block !important;
+        }
+        ^ .accordion-button {
+          display: inline-block;
+          border: none;
+          padding: 0;
+          margin: 8px 8px;
+          font-size: 20px;
+          overflow: hidden;
+          text-decoration: none;
+          text-align: left;
+          cursor: pointer;
+          white-space: nowrap;
+        }
+        ^ .accordion-button:focus {
+          outline: 0;
+        }
+      `,
+       properties: [
+        {
+          name: 'accordionCardShowDict',
+          value: {}
+        },
+        {
+          class: 'Boolean',
+          name: 'accordionCardShow',
+          value: true
+        },
+        {
+          class: 'String',
+          name: 'menuName',
+        }
+      ],
+       methods: [
+        function initE() {
+          var Menu = this.Menu;
```

Comment:
I would prefer you not do this since when I read the code below I wonder what Menu is. It looks like a global variable since it isn't scoped and it's capitalized. If I read it as this.Menu then I would know it was a required class and not be confused. It's only being accessed twice so this approach is also actually more characters.